ubuntudesign/cn.ubuntu.com
# add new news story - 和Ubuntu Core的一场黑马 - 将物联网的进化革命进行到底

## Done

Added the new news story: 和Ubuntu Core的一场黑马 - 将物联网的进化革命进行到底

## QA

 - Go to /news
 - Click on the link `和Ubuntu Core的一场黑马 - 将物联网的进化革命进行到底`
 - See that the page renders well across viewports
 - Check against the [copydoc](https://docs.google.com/document/d/1IKKFjlr4FKkP8lwgyy33fttKmQwsy0UoKDU9-JaPcxc/edit?ts=5850347d)
